clang/test/3C/add_struct_init.c

```diff
@@ -0,0 +1,32 @@
+// RUN: rm -rf %t*
+// RUN: 3c -base-dir=%S -alltypes -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
+// RUN: 3c -base-dir=%S -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
+// RUN: 3c -base-dir=%S -alltypes -addcr %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
+// RUN: 3c -base-dir=%S -alltypes -output-dir=%t.checked %s --
+// RUN: 3c -base-dir=%t.checked -alltypes %t.checked/add_struct_init.c -- | diff %t.checked/add_struct_init.c -
+
+
+struct foo {};
+struct bar { int *a; };
+//CHECK: struct bar { _Ptr<int> a; };
+struct fiz { int *a; };
+struct buz { struct bar a; };
+struct baz { struct buz a; };
+struct fuz { struct baz a; struct fiz b; };
+
+void test() {
```

Comment:
There aren't any nested structs that don't need an initializer. On the other hand, is it bad to add those? Can we add initializers to every struct and reduce our code size?